opm/autodiff/FlowMainEbos.hpp

```diff
@@ -728,8 +728,7 @@ namespace Opm
                                            FluidSystem::enableDissolvedGas(),
                                            FluidSystem::enableVaporizedOil(),
                                            eclState(),
-                                           *output_writer_,
-                                           defunctWellNames()));
```

Comment:
Also remove the defunctWellNames() member, it does not seem to be called anymore.

Reply:
Fixed in new PR